snmp/tests/common.py

```diff
@@ -49,6 +49,11 @@
 
 UNSUPPORTED_METRICS = [{'OID': "1.3.6.1.2.1.25.6.3.1.5.1", 'name': "IAmString"}]  # String (not supported)
 
+CAST_METRICS = [
+    {'OID': "1.3.6.1.4.1.2021.10.1.3.1", 'name': "cpuload1"},  # Counter32
```

Comment:
```suggestion
    {'OID': "1.3.6.1.4.1.2021.10.1.3.1", 'name': "cpuload1"},  # OctetString, Counter32
```

Reply:
Changed it, the Counter32 comment was wrong.